src/backend/storage/lmgr/proc.c

```diff
@@ -442,6 +442,7 @@ InitProcess(void)
 	Assert(pg_atomic_read_u32(&MyProc->clogGroupNext) == INVALID_PGPROCNO);
 
 	pg_atomic_init_u64(&MyProc->assignedXidCsn, InProgressXidCSN);
```

Comment:
Could you add `/* Remotexact */` above this line so we can find it easier later?

Reply:
Done.